Makefile

```diff
@@ -91,7 +92,7 @@ $(STABLE_TARGETS) $(STAGING_TARGETS): $(TMPDIR)/.helm/repository/local/index.yam
 	@mkdir -p $(shell dirname $@)
 	$(eval PACKAGE_SRC := $(shell echo $@ | sed 's@gh-pages/\(.*\)-[v0-9][0-9.]*.tgz@\1@'))
 	$(eval UNPACKED_TMP := $(shell mktemp -d))
-	$(HELM) --home $(TMPDIR)/.helm package $(PACKAGE_SRC) -d $(shell dirname $@)
+	$(HELM) package $(PACKAGE_SRC) -d $(shell dirname $@)
```

Comment:
It would help to explain (somewhere, not necessarily here) that  the `$(HELM)` call is going to use the env variables "exported" on L28-L30.